broker/lib/middleware.js

```diff
@@ -44,6 +46,38 @@ exports.validateCreateRequest = function () {
   };
 };
 
+exports.validateSchemaForRequest = function (target, operation) {
+  return function (req, res, next) {
+    const plan = getPlanFromRequest(req);
+
+    const schema = _.get(plan, `schemas.${target}.${operation}.parameters`);
```

Comment:
Can you please update/check-in local config [`settings.yml`](https://github.com/cloudfoundry-incubator/service-fabrik-broker/blob/master/broker/config/settings.yml) with schema definition.

Reply:
I have added it to the settings